docs/howto/multiple-storage-backends.md

```diff
@@ -0,0 +1,264 @@
+    <li><a href="#hybrid">Hybrid</a></li>
+  </ul>
+  
+  <div markdown="1" id="on-prem">
+
+This example setup configures lakeFS to manage data across two separate MinIO instances:
+
+```yaml
+blockstores:
+  signing:
+    secret_key: "some-secret"
+    stores:
+      - id: "minio-prod"
+        description: "Primary on-prem MinIO storage for production data"
+        type: "s3"
+        s3:
+          force_path_style: true
+          endpoint: 'http://minio-prod.local'
+          discover_bucket_region: false
+          credentials:
+            access_key_id: "prod_access_key"
+            secret_access_key: "prod_secret_key"
+      - id: "minio-backup"
+        description: "Backup MinIO storage for disaster recovery"
+        type: "s3"
+        s3:
+          force_path_style: true
+          endpoint: 'http://minio-backup.local'
+          discover_bucket_region: false
+          credentials:
+            access_key_id: "backup_access_key"
+            secret_access_key: "backup_secret_key"
+```
+
+  </div>
+
+  <div markdown="2" id="multi-cloud">
+
+This example setup configures lakeFS to manage data across two public cloud providers: AWS and Azure:
+
+```yaml
+blockstores:
+    signing:
+      secret_key: "some-secret"
+    stores:
+        - id: "s3-prod"
+          description: "AWS S3 storage for production data"
+          type: "s3"
+          s3:
+            region: "us-east-1"
+        - id: "azure-analytics"
+          description: "Azure Blob storage for analytics data"
+          type: "azure"
+          azure:
+            storage_account: "analytics-account"
+            storage_access_key: "EXAMPLE45551FSAsVVCXCF"
+
+```
+  </div>
+
+  <div markdown="3" id="hybrid">
+
+This hybrid setup allows lakeFS to manage data across both cloud and on-prem storages.
+```yaml
+blockstores:
+    signing:
+      secret_key: "some-secret"
+    stores:
+        - id: "s3-archive"
+          description: "AWS S3 storage for long-term archival"
+          type: "s3"
+          s3:
+            region: "us-west-2"
+        - id: "minio-fast-access"
+          description: "On-prem MinIO for high-performance workloads"
+          type: "s3"
+          s3:
+            force_path_style: true
+            endpoint: 'http://minio.local'
+            discover_bucket_region: false
+            credentials:
+              access_key_id: "minio_access_key"
+              secret_access_key: "minio_secret_key"
+
+```
+  </div>
+</div>
+
+### Key Considerations
+
+* Unique Blockstore IDs: Each storage backend must have a unique id.
+* Persistence of Blockstore IDs: Once defined, an id must not change.
+* S3 Authentication Handling:
+    * All standard S3 authentication methods are supported.
+    * If static credentials are provided, lakeFS will use them. Otherwise, it will fall back to the AWS credentials chain. 
+      This means that for setups with multiple storages of type `s3`, static credentials are required for all but one.
```

Comment:
Maybe we want to caveat that with the usage of `profiles`
Not necessarily defined as static but still can be used to define multiple aws blockstores